src/page/Home.tsx

```diff
@@ -88,7 +95,158 @@ class Home extends React.Component<{ history: any }> {
         }
     }
 
+    private overlay: any;
+
     public render() {
+        const searchExample = (
```

Comment:
Should we add a new functional component called `SearchExample` to make the render function shorter?